templates/resources/secrets_sync_azure_vault_service_principal.md.tmpl

```diff
@@ -0,0 +1,16 @@
+---
+page_title: "doppler_secrets_sync_azure_vault_service_principal Resource - terraform-provider-doppler"
+subcategory: ""
+description: |-
+	Manage an Azure Vault Doppler sync.
+---
+
+# doppler_secrets_sync_azure_vault_service_principal (Resource)
+
+Manage an Azure Vault Doppler sync.
```

Comment:
nit/optional: This template doesn't include "(Service Principal)" like in other places

Reply:
Per our discussion, we're going with a single `doppler_secrets_sync_azure_vault` resource since you can use that with both oauth and service principal Azure Vault integrations.